tensorflow/lite/python/lite.py

```diff
@@ -317,7 +317,8 @@ def from_concrete_functions(cls, funcs):
 
     Args:
       funcs: List of TensorFlow ConcreteFunctions. The list should not contain
-        duplicate elements.
+        duplicate elements. Currently converter can only convert a single 
+        ConcreteFunction. Converting multiple functions is under development."
```

Comment:
nit: Remove the quotation mark at the end.

Reply:
@gargn nicely spotted, seems like it was copied from the original place - now removed.